tests/functional/reporting-auth.js

```diff
@@ -39,6 +39,68 @@ define(
             assert.include(text, "Report as"); //Report as FooUser (logged in)
           })
           .end();
+      },
+
+      "Show double feature - buglist shows on focus, hides on blur": function() {
+        return FunctionalHelpers.openPage(
+          this,
+          url("/issues/new"),
+          ".wc-ReportForm-actions-button"
+        )
+          .findById("url")
+          .click()
+          .type("https://test.")
+          .sleep(500)
+          .end()
+          .findById("description")
+          .click()
+          .end()
+          .sleep(9500)
+          .findById("bugList")
+          .getVisibleText()
+          .then(function(text) {
+            assert.equal(text, "");
+          });
+      },
+
+      "Show double feature - 3 issues with url found": function() {
+        return FunctionalHelpers.openPage(
+          this,
+          url("/issues/new"),
+          ".wc-ReportForm-actions-button"
+        )
+          .findById("url")
+          .click()
+          .type("https://webctest.")
+          .sleep(500)
+          .end()
+          .findById("bugList")
+          .findAllByTagName("li a")
+          .getVisibleText()
+          .then(function(texts) {
+            assert.isAtLeast(texts.length, 3, "found 3 issues");
+          })
+          .end();
+      },
+
+      "Show double feature - no issue with url found": function() {
+        return FunctionalHelpers.openPage(
+          this,
+          url("/issues/new"),
+          ".wc-ReportForm-actions-button"
+        )
+          .findById("url")
+          .click()
+          .type("https://webctest1234567890")
+          .sleep(500)
+          .end()
+          .findById("bugList")
+          .findAllByTagName("li a")
+          .getVisibleText()
+          .then(function(texts) {
+            assert.isBelow(texts.length, 1, "works");
```

Comment:
If you expect 0 results, can we do `assert.equal(text.length, 0, "No results returned")`?

("works also seems like leftover debug text, can we have it be more descriptive so future failures are less confusing?)

Reply:
+1